corehq/apps/hqwebapp/static/hqwebapp/js/ckeditor_knockout_bindings.js

```diff
@@ -9,42 +8,45 @@ hqDefine('hqwebapp/js/ckeditor_knockout_bindings', [
     'underscore',
     'knockout',
     'hqwebapp/js/initial_page_data',
+    'ckeditor5/ckeditor5.js',
```

Comment:
Why does this one have a `.js` extension while others do not?

Reply:
Lost a comment during an earlier rebase. Added back in https://github.com/dimagi/commcare-hq/pull/35498/commits/218e3c10c81ad1eaa2f1e16fddaa5589939d4337#diff-baa0904f149ab075d59892a3e08593e3e48db2992968850384367903328cdeceR11

There are other `ckeditor.foo` files in the same directory.